components/custom-tab-bar.tsx

```diff
@@ -0,0 +1,195 @@
+import MenuIcon from '@/assets/icons/svg/menu-icon.svg'
+import MoviesIcon from '@/assets/icons/svg/movies-icon.svg'
+import PlayIcon from '@/assets/icons/svg/play-icon.svg'
+import TicketsIcon from '@/assets/icons/svg/tickets-icon.svg'
+import { BottomTabBarProps } from '@react-navigation/bottom-tabs'
+import { LinearGradient } from 'expo-linear-gradient'
+import { Pressable, StyleSheet, Text, View } from 'react-native'
+
+const CustomTabBar = ({
+  state,
+  descriptors,
+  navigation,
+}: BottomTabBarProps) => {
+  const tabConfig = [
+    {
+      name: 'Movies',
+      icon: MoviesIcon,
+    },
+    {
+      name: 'Trailers',
+      icon: PlayIcon,
+    },
+    {
+      name: 'Tickets',
+      icon: TicketsIcon,
+    },
+    {
+      name: 'Menu',
+      icon: MenuIcon,
+    },
+  ]
+
+  const renderIcon = (
+    IconComponent: any,
+    size: number,
+    color: string,
+    index: number
+  ) => {
+    const Icon = IconComponent
+    let width, height
+
+    if (index === 0) {
+      // Movies
+      width = 24
+      height = 20
+    } else if (index === 1) {
+      // Play/Trailers
+      width = 28
+      height = 18
+    } else if (index === 2) {
+      // Tickets
+      width = 33.5
+      height = 24.7
+    } else {
+      // Menu
+      width = 25
+      height = 20
+    }
+
+    return <Icon width={width} height={height} color={color} />
+  }
+
+  return (
+    <View style={styles.tabBar}>
+      {state.routes.map((route, index) => {
+        const { options } = descriptors[route.key]
+        const isFocused = state.index === index
+
+        const onPress = () => {
+          const event = navigation.emit({
+            type: 'tabPress',
+            target: route.key,
+            canPreventDefault: true,
+          })
+
+          if (!isFocused && !event.defaultPrevented) {
+            navigation.navigate(route.name)
+          }
+        }
+
+        const tab = tabConfig[index]
+
+        return (
+          <Pressable
+            key={route.key}
+            onPress={onPress}
+            style={[
+              styles.tabItem,
+              isFocused && styles.tabItemActive,
+              index > 0 && styles.tabItemSpacing,
+            ]}
+          >
+            {!isFocused ? (
+              <View style={styles.gradientBackground}>
+                {/* Primer gradiente: vertical sólido #D9D9D9 */}
+                <LinearGradient
+                  colors={['#D9D9D9', '#D9D9D9']}
+                  start={{ x: 0, y: 0 }}
+                  end={{ x: 0, y: 1 }}
+                  style={styles.gradientLayer1}
+                />
+                {/* Segundo gradiente: diagonal #1F1F1F a #333333 */}
+                <LinearGradient
+                  colors={['#1F1F1F', '#333333']}
```

Comment:
The color of the gradients should depend on the theme, check how it looks in light theme